reflex/components/component.py

```diff
@@ -598,6 +598,19 @@ def create(cls, *children, **props) -> Component:
         # Import here to avoid circular imports.
         from reflex.components.base.bare import Bare
 
+        # Translate deprecated props to new names.
+        known_props = cls.get_props()
```

Comment:
nit: this can be `known_props = [prop for prop in cls.get_props() if prop in ["type", ...]]`